tappsbt/interface.go

```diff
@@ -750,3 +759,19 @@ func deserializeTweakedScriptKey(pOut psbt.POutput) (*asset.TweakedScriptKey,
 		Tweak:  tweak,
 	}, nil
 }
+
+// Encode encodes the virtual packet into a byte slice.
+func Encode(vPkt *VPacket) ([]byte, error) {
```

Comment:
Kinda surprised we didn't have something like this already? Perhaps we should have the encode be a method to take a writer? Same below w/ a reader.

Reply:
Ah I see below just uses the decode that takes a reader.